modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCTestSupport.java

```diff
@@ -255,6 +260,22 @@ protected void assertAttributeValuesEqual(Object expected, Object actual) {
         assertEquals(expected, actual);
     }
 
+    /**
+     * Returns the WGS84 {@link CoordinateReferenceSystem} with an axis order taking into account
+     * the {@link #forceLongitudeFirst} setting.
+     */
+    protected CoordinateReferenceSystem getWGS84() throws FactoryException {
```

Comment:
This is a confusing name, ``DefaultGeographicCRS.WGS84`` is a well-known GeoTools constant with axis order in lon/lat order. Best to use ``decodeEPSG(4326)`` directly instead, as it leaves the reader with the doubt about the CRS (and thus offers incentive to go and read the Javadoc).